src/main/java/eu/europeana/metis/sandbox/config/SandboxConfig.java

```diff
@@ -24,12 +31,30 @@
   @Value("${sandbox.dataset.creation.threads.thread-prefix}")
   private String threadPrefix;
 
+  @Autowired
+  private ResourceLoader resourceLoader;
```

Comment:
It is recommended to not use field injection, but rather @autowired constructors. This is so that you can mock them better.